src/core/thread/topology.hpp

```diff
@@ -80,8 +80,12 @@ class Neighbor
         kStateLinkRequest,               ///< Sent a MLE Link Request message
         kStateValid,                     ///< Link is valid
     };
-
+    
+#if _WIN32
```

Comment:
I had to do this because of a weird issue when built for Windows. When reading this value, it was being treated as a signed int, so all enum values >= 4, the comparisons to the enum and mState would fail. If someone has a better solution for this, let me know.